packages/story-utils/src/generateSnapshots.ts

```diff
@@ -53,23 +53,23 @@ export default function generateSnapshots(
     test(`${storyName} story renders snapshot`, async () => {
       const view = render(createElement(Story));
 
+      // When components that include Apollo's useQuery are rendered we need
+      // to await an act that pushes the test to the end of the event loop.
+      // https://trojanowski.dev/apollo-hooks-testing-without-act-warnings/
+      await wait();
+
       // @storybook/testing-react doesn't run play functions automatically (as of v1.0.0). So if
       // one is present, run it before taking a snapshot.
       if (Story.play) {
         const storyContext = {
           loaded: {},
           abortSignal: new AbortController().signal,
-          canvasElement: document.createElement('div'),
+          canvasElement: view.container,
```

Comment:
I was passing an empty div into here... and then wondering why play functions couldn't find anything 😆